docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -9,7 +9,7 @@ Quarkus Security is a framework that provides the architecture, multiple authent
 
 Before you start building security into your Quarkus applications, review the overview information to learn about the Quarkus Security architecture and the different authentication and authorization mechanisms that Quarkus supports.
 
-To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-authentication-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC). 
+To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-authentication-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC).
```

Comment:
This is probably outside the scope of the PR but should the `the JPA provider` be `a JPA provider` or `your JPA provider`?